README.md

```diff
@@ -193,3 +193,27 @@ If you register the service providers in the correct order in your container (A
 - [Simplex](https://github.com/mnapoli/simplex): A [Pimple 3](https://github.com/silexphp/Pimple) fork with full [container-interop](https://github.com/container-interop/container-interop) compliance and cross-framework service-provider support.
 - [Service provider bridge bundle](https://github.com/thecodingmachine/service-provider-bridge-bundle): Use container-interop's service-providers into a Symfony container.
 
+##FAQ
+
+####Why static methods?
```

Comment:
Indeed, it's not valid CommonMark: http://spec.commonmark.org/0.24/#atx-heading
I'll fix it later.